tailhead/__init__.py

```diff
@@ -223,11 +223,17 @@ def head(self, lines=10):
         """
         Return the top lines of the file.
```

Comment:
Could you also expend the docstring to document new behavior?

Reply:
Oops, I forgot about documentation. I added it here and in the user interface below. Also added a new example in README.rst

Reply:
I have added the documentation you suggested and an example in README.rst. Please take a look.

Reply:
@Kentzo could you please take a look at the documentation added?